### ISSUE TITLE
what does the 'wb = 0.00001' mean in mx.model.FeedForward

### ISSUE THREAD

**Issue by albertyou2** [dmlc/mxnet#3007]
I am confusing about this arg in  mx.model.FeedForward.

**Comment by Godricly** [comment]
Are you talking about the weight decay? which file are you looking at?

**Comment by albertyou2** [comment]
@Godricly  You can see train_model.py in example/image-classification at this line :
 model = mx.model.FeedForward(
        ctx                = devs,
        symbol             = network,
        num_epoch          = args.num_epochs,
        learning_rate      = args.lr,
        momentum           = 0.9,
        wd                 = 0.00001,
        initializer        = mx.init.Xavier(factor_type="in", magnitude=2.34),
        **model_args)

**Comment by Godricly** [comment]
I did check that. U made a typo on the issue title which is misleading. It should be the weight decay parameter. U can find out how it is used in [python/mxnet/optimizer.py](https://github.com/dmlc/mxnet/blob/master/python/mxnet/optimizer.py).

**Comment by albertyou2** [comment]
@Godricly Thank you ,I got it
